jsonnet/rules.jsonnet

```diff
@@ -66,6 +66,15 @@
           },
         ],
       },
+      {
+        name: 'openshift-sre.rules',
+        rules: [
+          {
+            expr: sum(rate(apiserver_request_count{job="apiserver"}[10m])) by (code)
```

Comment:
both of these need to be quoted I believe (at least CI seems to be complaining)

Reply:
yup I see I missed the formatting completely !